runtime/reconcilers/project_parser.go

```diff
@@ -264,6 +265,10 @@ func (r *ProjectParserReconciler) reconcileParser(ctx context.Context, inst *dri
 				continue
 			}
 			r.C.Logger.Warn("Parser error", zap.String("path", e.FilePath), zap.String("error", e.Message))
+			debug := os.Getenv("DEBUG")
+			if debug == "1" {
+				fmt.Println("Parser error", e.Message)
+			}
```

Comment:
Remove

Reply:
I wonder if uncommenting ` // logger, err := zap.NewDevelopment() ` will show this error too?